### DISCUSSION TITLE
Error "Your repo passphrase was stored in a password manager which is no longer available" after updating to 0.7.3

### DISCUSSION THREAD

**Discussion by mauromol** [borgbase/vorta#876] [General]
**Describe the bug**
Today I updated Vorta Backup to 0.7.3 (from 0.7.2). Now my backup jobs fail with the error:
"Your repo passphrase was stored in a password manager which is no longer available".

**To Reproduce**
1. open Vorta
2. click on "Start backup"

Or just wait for a scheduled backup to start and see that it won't create any new archive in "Archives" list.

**Environment (please complete the following information):**
- OS: Kubuntu 20.04
- Vorta version: 0.7.3
- Installed from: Vorta PPA

**Additional context**

Logs:

```
2021-02-19 19:00:00,001 - apscheduler.executors.default - INFO - Running job "VortaScheduler.create_backup (trigger: cron[hour='*/1', minute='0'], next run at: 2021-02-19 20:00:00 CET)" (scheduled at 2021-02-19 19:00:00+01:00)
2021-02-19 19:00:00,006 - vorta.scheduler - INFO - Starting background backup for Home su NAS OpenMediaVault
2021-02-19 19:00:00,009 - vorta.notifications - DEBUG - success notifications suppressed
2021-02-19 19:00:00,010 - vorta.borg.borg_thread - DEBUG - Using VortaKWallet5Keyring keyring to store passwords.
2021-02-19 19:00:00,066 - vorta.borg.borg_thread - DEBUG - Password not found in primary keyring. Falling back to VortaDBKeyring.
2021-02-19 19:00:00,066 - vorta.scheduler - ERROR - Conditions for backup not met. Aborting.
2021-02-19 19:00:00,067 - vorta.scheduler - ERROR - Your repo passphrase was stored in a password manager which is no longer available.
Try unlinking and re-adding your repo.
2021-02-19 19:00:00,067 - vorta.notifications - DEBUG - notification not suppressed
2021-02-19 19:00:00,120 - apscheduler.executors.default - INFO - Job "VortaScheduler.create_backup (trigger: cron[hour='*/1', minute='0'], next run at: 2021-02-19 20:00:00 CET)" executed successfully
```

**Comment by mauromol** [comment]
Please note that in KWallet I do have vorta-repo among the list of connected applications:

![immagine](https://user-images.githubusercontent.com/1742494/108545428-111ca080-72e8-11eb-94a4-448d462634fb.png)

**Comment by ajvalenc** [comment]
Same error with the 0.7.3 update (Manjaro KDE). I tried unlinking and re-adding the repo but didn't work. I got "ERROR: Connection closed by remote host. Is Borg working on the server?"

**Comment by m3nu** [comment]
The code to select the password is here. Feel free to see, if something could be improved or why it's not working for you: [vorta/borg/borg_thread.py#L141](https://github.com/borgbase/vorta/blob/master/src/vorta/borg/borg_thread.py#L141)

Basically it will try to save it to the system keychain first, but also fall back to saving it to the settings sqlite database.

**Comment by mauromol** [comment]
Hmmm... not sure why this is a discussion rather than an issue, perhaps it was my fault when I opened the bug report?

I do not have the skill necessary to understand that code, what I can say for sure is that it started to happen after updating Vorta to 0.7.3. This did not happen when I updated to 0.7.2 some weeks ago.
Unlinking and re-adding the repo worked, however it's rather a workaround.

**Reply by m3nu** [reply]
> Hmmm... not sure why this is a discussion rather than an issue, perhaps it was my fault when I opened the bug report?

Can be hard to tell. Basically issues are planned code changes in Vorta. Some discussions result in an issue, but many don't. Before we just had a lot of issues that were really discussions about packaging. Those make it very hard for me to prioritize actual issues. So a discussion is preferred to verify if there is an *issue* that can be addressed within Vorta.

Discussion > Issue+Milestone > Pull Request > Release

**Comment by mauromol** [comment]
Seems like I'm not the only one with this problem: see #859, #888, #879.

I now see there's a pending update in the PPA repository (new version is 0.7.3-0ubuntu0ppa2, while the old one was 0.7.3-0ubuntu0): what's the changelog? Is it meant to contain a fix for this problem?

Also I see now that I indeed opened this as an issue (#871) and was later converted into a discussion. GitHub notifications didn't help me to understand this.

**Reply by m3nu** [reply]
Most discussion on this is [here](https://github.com/borgbase/vorta/discussions/859#discussioncomment-401271) now. You can try the current `master` branch to see if your issue is resolved there.

**Reply by mauromol** [reply]
Unfortunately I can't test any more, because I unlinked and re-added my repo. I was just curious now to know what is the changelog for the new 0.7.3-0ubuntu0ppa2 version, since the deb package does not carry a changelog.
